### ISSUE TITLE
Implement pre calculated formulas.

### ISSUE THREAD

**Issue by AirSeeker** [codemix/yii2-excelexport#7]
In case when in cell insert formula it won't be calculated until user enter edit mode. On read mode, values in calculated cells will be 0.
To allow calculation on read mode, before save at writer should be used setPreCalculateFormulas() method.
By default preCalculateFormulas value should be false, i think.
For example if in configs object 'preCalculateFormulas' => true, than formulas will be calculated.

            public $preCalculateFormulas = false;
            $writer = new $class($workbook);
            if($this->preCalculateFormulas){
                $writer->setPreCalculateFormulas();
            }
            $writer->save((string) $this->getTmpFile());

**Comment by mikehaertl** [comment]
@AirSeeker Instead of adding such specific options, would it help if we expose the Excel writer object via a getter? So before calling `send()` or `saveAs()` you could fetch the object and do any additional modification you like:

```
use codemix\excelexport;
$file = new ExcelFile([ ... ]);
$file->getWriter()->setPreCalculateFormulas();
$file->saveAs('myfile.xlsx');
```

**Comment by AirSeeker** [comment]
@mikehaertl Yes, it would be more helpful, make getter for writer.

**Comment by mikehaertl** [comment]
Ok, I can add it. As we already have a public property `$writer` for the writer class name this needs to be renamed if the we add `getWriter()`. So we have a BC breaking change which requires a major version bump. Not very nice, but I see no better way unless we call it `getExcelWriter()` which is not very nice either.

**Comment by mikehaertl** [comment]
@AirSeeker Implemented in latest code on master but I had no time to test it. Could you help here, please? You only need to update to the latest ExcelFile from here: https://github.com/codemix/yii2-excelexport/blob/master/src/ExcelFile.php

**Comment by AirSeeker** [comment]
@mikehaertl testing passed successfully. Thanks for your time.

**Comment by mikehaertl** [comment]
Released 2.0.0. Thanks.
